### PR TITLE
Scroll

### PR DESCRIPTION
**Pull request by kel29**
adjusts chat window view heights, implements scrolling in chats and topics

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,29 +2,25 @@
   text-align: center;
 }
 
-.navbar {
```
Comment (renburnett) [inline src/App.css:5]: lol forgot we still had this

File: src/components/OtherUserMessages.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const OtherUserMessages = (props) => {
```
Comment (renburnett) [inline src/components/OtherUserMessages.js:3]: nice idea to make this a component

File: src/containers/MessageForm.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ class MessageForm extends PureComponent {
 
   render () {
     return (
-      <form className="row message-form form-inline" onSubmit={this.handleSubmit}>
+      <form className="row message-form form-inline card-footer" onSubmit={this.handleSubmit}>
```
Comment (renburnett) [inline src/containers/MessageForm.js:49]: woo!

**Review by renburnett — APPROVED** [review]
`<div className='bard-body overflow-auto'>` hehehe

File: src/App.css [hunk 4]
```diff
@@ -15,14 +11,22 @@
...
 
-.form-submit-btn {
-  width: 20%;
-}
-
 .message-body {
   font-size: 1.4rem;
 }
 
+.selected-topic {
```
Comment (ianhunterharold) [inline src/App.css:18]: style master.

**Review by ianhunterharold — APPROVED** [review]

**Pull request merged**